src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,17 @@
-@mixin hover($_property, $_toValue) {
-  transition: #{$_property} 0.3s;
-  &:hover {
-    #{$_property}: $_toValue;
+@mixin tablet {
```

Comment:
The 'tablet' mixin uses a min-width of 768px. This is not always the correct breakpoint for tablets. It's better to use a variable for this and adjust as needed per project basis.